### PR TITLE
Drop ETH, BCH, LTC payments

### PR DESCRIPTION
**Pull request by g4s8**
#1304 - throw exception for not supported paymant methods

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1653](https://github.com/zerocracy/farm/issues/1653) is now in scope, role is `REV`

File: src/test/java/com/zerocracy/pmo/PeopleTest.java [hunk 1]
```diff
@@ -622,6 +596,17 @@ private void failsWallet(final String bank, final String wallet)
         people.wallet("yegor512", bank, wallet);
     }
 
+    private void doesnSupportWallet(final String bank, final String name)
+        throws IOException {
+        final FkFarm farm = new FkFarm(new FkProject());
+        final People people = new People(farm).bootstrap();
+        this.thrown.expect(SoftException.class);
+        this.thrown.expectMessage(
+            String.format("We doesn't support %s wallets", name)
```
Comment (krzyk) [inline src/test/java/com/zerocracy/pmo/PeopleTest.java]: @g4s8 `We doesn't` -> `We don't`

File: src/test/java/com/zerocracy/pmo/PeopleTest.java [hunk 2]
```diff
@@ -622,6 +596,17 @@ private void failsWallet(final String bank, final String wallet)
         people.wallet("yegor512", bank, wallet);
     }
 
+    private void doesnSupportWallet(final String bank, final String name)
+        throws IOException {
+        final FkFarm farm = new FkFarm(new FkProject());
```
Comment (krzyk) [inline src/test/java/com/zerocracy/pmo/PeopleTest.java]: @g4s8 `farm` and `people` can be inlined

File: src/main/java/com/zerocracy/pmo/People.java [hunk 3]
```diff
@@ -409,23 +409,14 @@ public void wallet(final String uid, final String bank,
...
+        if ("eth".equals(bank)) {
+            throw new SoftException("We doesn't support Ethereum wallets");
         }
-        if ("ltc".equals(bank)
-            && !wallet.matches("[0-9a-zA-Z]{34}")) {
-            throw new SoftException(
-                new Par("Litecoin address is not valid: `%s`").say(wallet)
-            );
+        if ("ltc".equals(bank)) {
+            throw new SoftException("We doesn't support Litecoin wallets");
```
Comment (krzyk) [inline src/main/java/com/zerocracy/pmo/People.java]: @g4s8 `We doesn't` -> `We don't`

File: src/main/java/com/zerocracy/pmo/People.java [hunk 4]
```diff
@@ -409,23 +409,14 @@ public void wallet(final String uid, final String bank,
...
+        if ("bch".equals(bank)) {
+            throw new SoftException("We doesn't support Bitcoin Cash wallets");
         }
-        if ("eth".equals(bank)
-            && !wallet.matches("(0x)?[0-9a-fA-F]{40}")) {
-            throw new SoftException(
-                new Par("Ethereum address is not valid: `%s`").say(wallet)
-            );
+        if ("eth".equals(bank)) {
+            throw new SoftException("We doesn't support Ethereum wallets");
```
Comment (krzyk) [inline src/main/java/com/zerocracy/pmo/People.java]: @g4s8 `We doesn't` -> `We don't`

File: src/main/java/com/zerocracy/pmo/People.java [hunk 5]
```diff
@@ -409,23 +409,14 @@ public void wallet(final String uid, final String bank,
                 new Par("Bitcoin address is not valid: `%s`").say(wallet)
             );
         }
-        if ("bch".equals(bank)
-            && !wallet.matches("[pq][a-z0-9]{41}")) {
-            throw new SoftException(
-                new Par("Bitcoin Cash address is not valid: `%s`").say(wallet)
-            );
+        if ("bch".equals(bank)) {
+            throw new SoftException("We doesn't support Bitcoin Cash wallets");
```
Comment (krzyk) [inline src/main/java/com/zerocracy/pmo/People.java]: @g4s8 `We doesn't` -> `We don't`

**Comment by g4s8** [comment]
@krzyk thanks, please look again

**Comment by krzyk** [comment]
@g4s8 thanks

**Comment by krzyk** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @krzyk[/z](https://www.0crat.com/u/krzyk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1653](https://github.com/zerocracy/farm/issues/1653) is now out of scope

**Comment by ypshenychka** [comment]
@krzyk According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

All found issues are cosmetic. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by krzyk** [comment]
@ypshenychka I confirm

**Comment by ypshenychka** [comment]
@krzyk thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
> @0crat quality acceptable ([here](https://github.com/zerocracy/farm/issues/1653#issuecomment-416462157))

@ypshenychka There is no quality review for [#1653](https://github.com/zerocracy/farm/issues/1653), no performer

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**